images/ose-baremetal-installer.yml

```
@@ -35,5 +35,6 @@ owners:
 - kni-devel-deployment@redhat.com
 konflux:
   vm_override:
-    aarch64: linux-cxlarge/arm64
+    x86_64: linux-d160-m2xlarge/amd64
+    aarch64: linux-d160-m2xlarge/arm64
```

```suggestion
    x86_64: linux-c4xlarge/amd64
    aarch64: linux-c4xlarge/arm64
```
Since  this build might not require the 160 GB disk space. Suggesting to use this flavor which has `32` CPU, which would be a significant improvement.